tools/checked-c-convert/ProgramInfo.cpp

```diff
@@ -63,6 +67,9 @@ void PointerVariableConstraint::print(raw_ostream &O) const {
   for (const auto &I : vars) 
     O << "q_" << I << " ";
   O << " }";
+
+  if (FV)
+    FV->print(O);
```

Comment:
Probably worth parenthesizing the dumped FV.   This will avoid confusion in reading dumps when you have pointers to functions that return pointers and so on.